packages/block-library/src/gallery/editor.scss

```diff
@@ -8,7 +8,6 @@
 
 	// @todo: this deserves a refactor, by being moved to the toolbar.
 	.block-editor-media-placeholder {
-		margin-bottom: $grid-unit-15;
 		padding: $grid-unit-15;
```

Comment:
Turns out you can remove this padding as well, it's inherited by the placeholder component and no longer needed!